application.py

```diff
@@ -267,8 +229,9 @@ def autocomplete(search):
 def exchange_code_for_tokens(code):
     # Hit AWS Cognito auth endpoint with specific payload for exchange tokens.
     # This is the endpoint for the AWS Cognito user pool. It will not change.
+    AWS_ENV = os.getenv("AWS_ENVIRONMENT")
     token_url = (
-        "https://keh-tech-audit-tool.auth.eu-west-2.amazoncognito.com/oauth2/token"
+        f"https://tech-audit-tool-api-sdp-{AWS_ENV}.auth.eu-west-2.amazoncognito.com/oauth2/token"
```

Comment:
It would be consistent with other use cases (terraform etc) if you set AWS_ENV as either sdp-dev, sdp-prod, sdp-sandbox rather than just dev, prod, sandbox.  This way ytou can also remove the random sdp stirng from the end of the tool name:
  tech-audit-tool-api-sdp-{AWS_ENV}